cafebabel/templates/base.html

```diff
@@ -1,8 +1,40 @@
+{% from "_meta-macro.html" import meta with context %}
 <!doctype html>
 <meta charset=utf-8>
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
-<meta name=description content="{% block meta_description %}{% endblock %}">
+
+{% set description -%}
```

Comment:
Trying to put it within the base template adds extra complexity. I don't get why we try to put meta globally instead of per template/part?

Reply:
@davidbgk - the code 'll looked a bit like this?